### PR TITLE
Profiler is sometimes unresponsive

### LINKED ISSUE #6796: Game from bundle(debug)/(editor) not response(windows)
### Expected behaviour
1)Game run
### Actual behaviour
2)Game not response
### Steps to reproduce
1)Create new empty project
2)Project->build->game not response
2.1)Same for windows bundle(debug)
2.2)release bundle worked
<hr/>

<table>
<tr><td>Build time</td><td>2022-07-11T09:22:18.167890</td></tr>
<tr><td>Defold channel</td><td>editor-alpha</td></tr>
<tr><td>Defold editor sha</td><td>c0ed825ede46b0112d1e4fd744fd50f460073229</td></tr>
<tr><td>Defold engine sha</td><td>80b1b73fd9cdbd4682c2583403fddfbaf0919107</td></tr>
<tr><td>Defold version</td><td>1.3.4</td></tr>
<tr><td>GPU</td><td>AMD Radeon(TM) Graphics</td></tr>
<tr><td>GPU Driver</td><td>4.6.14756 Compatibility Profile Context 20.40.64 27.20.14064.3</td></tr>
<tr><td>Java version</td><td>11.0.15+10-LTS</td></tr>
<tr><td>OS arch</td><td>amd64</td></tr>
<tr><td>OS name</td><td>Windows 10</td></tr>
<tr><td>OS version</td><td>10.0</td></tr>
<table>

### PR DESCRIPTION
**Pull request by britzl**
Added a `profiler.sleep_between_server_updates` *game.project* setting to set the delay in milliseconds between profiler server updates. Try changing this value to something like 50ms if you have problems with the computer becoming unresponsive while using the profiler.

The profiler will now also reuse any existing profiler port when reconnecting. This should solve reported issues with profiler connection problems.

Fixes #6796

### CODE REVIEW

File: editor/resources/meta.edn [hunk 1]
```diff
@@ -811,6 +811,10 @@
    :help "enable CPU usage sampling in release"
    :default false
    :path ["profiler" "track_cpu"]}
+  {:type :integer
+   :help "Number of milliseconds to sleep between server updates"
+   :default 0
+   :path ["profiler" "sleep_between_server_updates"]}
```
Comment (AGulev) [inline editor/resources/meta.edn:817]: Could you pls add it into `meta.properties` as well?

Comment (britzl) [reply editor/resources/meta.edn:817]: Ah, yes, forgot!

**Comment by britzl** [comment]
Added option to not automatically start the profiler on engine startup. This is on by default and can be toggled from game.project:

```
profiler.autostart = 0 | 1
```

The profiler can manually be started using `profiler.start()`. I've played around with an option to also have a `profiler.stop()` but this may result in a crash in Remotery. Some notes on this here: https://github.com/Celtoys/Remotery/issues/231

**Comment by britzl** [comment]
I've reverted the code to manually start and stop the profiler. It turns out that it was the thread sampler that was causing the issues on Windows.

**Review by AGulev — APPROVED** [review]

**Pull request merged**